content/blog/customizing-hcl/index.md

```diff
@@ -0,0 +1,169 @@
+ADD https://your.git.url/cloudnativepg/-/blob/main/bake/files/etc/ldap/ldap.conf?ref_type=heads /etc/ldap/
+ADD https://your.git.url/cloudnativepg/-/blob/main/bake/files/usr/local/share/ca-certificates/EuropeanSSLServerCA2.crt?ref_type=heads /usr/local/share/ca-certificates/
+ADD https://your.git.url/cloudnativepg/-/blob/main/bake/files/usr/local/share/ca-certificates/RootCA1v0.crt?ref_type=heads /usr/local/share/ca-certificates/
+ADD https://your.git.url/cloudnativepg/-/blob/main/bake/files/usr/local/share/ca-certificates/SubCA1v1.crt?ref_type=heads /usr/local/share/ca-certificates/
+RUN update-ca-certificates 
+USER 26
+EOT
+  matrix = {
+    tgt = [
+      "myimage"
+    ]
+    pgVersion = [
+      "13.21",
+      "14.18",
+      "15.13",
+      "16.9",
+      "17.5",
+    ]
+  }
+  name = "postgresql-${index(split(".",cleanVersion(pgVersion)),0)}-standard-bookworm"
+  target = "${tgt}"
+  args = {
+    BASE_IMAGE = "ghcr.io/cloudnative-pg/postgresql:${cleanVersion(pgVersion)}-standard-bookworm",
+    EXTENSIONS = "${getExtensionsString(pgVersion, extensions)}",
+  }
+}
+```
+
+Starting at the beginning of the file:
+
+- The `environment` variable is set to `production` for all of my images, 
+because I use the same image to stage it through dev/test/prod.
+- The `registry` variable contains the repo upload url, so I don't have to add 
+this information every time I build an image.
+- The `platforms` variable is `linux/amd64` for all of my images.
+- The `extensions` variable contains some extensions I use regularly.
+- The `dockerfile-inline` part is extended with binaries, some of them are handy
+ to have, some needed by extensions or other tools I use e.g. [pgwatch](https://github.com/cybertec-postgresql/pgwatch).
+- With the `sed` command I add needed locales and build them.
+- With the `ADD` commands I extend the image with
+  - .psqlrc file, to have a nice psql Command-line even when connecting via 
+  `kubectl cnpg psql XXX`
+  - ldap.conf and the needed certs
+
+### Step 2: Build the image
+
+We can now build the image using the following command:
+
+```bash
+docker buildx bake -f docker-bake.hcl -f cwd://bake.hcl "https://github.com/cloudnative-pg/postgres-containers.git" myimage
+```
+
+### Step 3: Use it
+
+The only missing step to use the images is to update your 
+[Image Catalog / Cluster Image Catalog](https://cloudnative-pg.io/documentation/current/image_catalog/) 
+with the newly built images.
+Test them and stage them through your environment.
+
+## Conclusion
+
+Once you prepare the override file to fit to your needs, the only manual steps 
+to build new images are
+  - udpate the `pgVersion` variable
+  - run the `docker buildx bake` command
+   I hope this helps streamline your image customization process as much as it 
+   did mine—feel free to build on it, and share your own improvements too! 
+   You can find the supportive team in the CloudNativePG channels on the CNCF Slack workspace.
```

Comment:
with this spacing of the lines, I think the paragraph starting line 167 could become a code block.
The lines should not begin with spaces, right?
Also, to better signal what the rendered outcome should be, I suggest blank lines surrounding the two-item list.